Python/Tests/Analysis/AnalysisTest.cs

```diff
@@ -6860,10 +6860,10 @@ class Y:
                 new { Code="from ..hij import A", Index=10, Expected="abc.hij", Base="abc.deg.HIJ" },
             }) {
                 var entry = ProcessTextV3(item.Code);
-                var walker = new ImportedModuleNameWalker(item.Base, string.Empty, item.Index);
+                var walker = new ImportedModuleNameWalker(item.Base, string.Empty, item.Index, null);
                 entry.Modules[entry.DefaultModule].Tree.Walk(walker);
 
-                Assert.AreEqual(item.Expected, walker.ImportedModules.First());
+                Assert.AreEqual(item.Expected, walker.ImportedModules.First().Name);
```

Comment:
`FirstOrDefault()?.Name` - better to fail the test with `name != null` than InvalidOperationException